--- libwpd-0.9.0/libwpd-stream-0.9.pc.in~	2010-08-26 13:35:21.000000000 +0200
+++ libwpd-0.9.0/libwpd-stream-0.9.pc.in	2010-12-28 14:20:59.318039334 +0100
@@ -7,4 +7,4 @@
 Description: Sample Input stream implementation for libwpd, a library for reading and writing Corel WordPerfect(tm) documents
 Version: @VERSION@
 Libs: -L${libdir} -lwpd-stream-@WPD_MAJOR_VERSION@.@WPD_MINOR_VERSION@
-Cflags: -I${includedir}/libwpd-@WPD_MAJOR_VERSION@.@WPD_MINOR_VERSION@
+Cflags: -I${includedir}/libwpd-@WPD_MAJOR_VERSION@.@WPD_MINOR_VERSION@ -I${includedir}/libwpd-@WPD_MAJOR_VERSION@.@WPD_MINOR_VERSION@/libwpd-stream
